torch/_torch_docs.py

```diff
@@ -2223,9 +2223,9 @@ def merge_dicts(*dicts):
     tensor([[ 2.4112, -0.7486,  1.4551],
             [-0.7486,  1.3544,  0.1294],
             [ 1.4551,  0.1294,  1.6724]])
-    >>> a = torch.randn(3, 2, 2)
+    >>> a = torch.randn(3, 2, 2) # Example for batched input
     >>> a = a @ a.mT + 1e-03 # make symmetric positive-definite
-    >>> l = torch.cholesky(a)
+    >>> l = torch.linalg.cholesky(a)
```

Comment:
nit: we should change this to `torch.cholesky` too since this is a docstring for the `torch.cholesky` function.

Reply:
I reflected it. It is another my mistake.